addons/website_slides/static/lib/pdfslidesviewer/PDFSlidesViewer.js

```diff
@@ -117,6 +124,29 @@ var PDFSlidesViewer = (function(){
         return this.queueRenderPage(this.pdf_page_current);
     };
 
+    /*
+     * Calculate a scale to fill as much as possible of the available width,
+     * without making the document overtly long.
+     */
+    PDFSlidesViewer.prototype.getDefaultScale = function(page) {
+        var self = this;
+        var maxWidth = this.canvas.parentNode.clientWidth;
+        var maxHeight = this.canvas.parentNode.clientHeight;
+        var hScale = maxWidth / page.view[2];
+        var vScale = maxHeight / page.view[3];
+        return Math.min(hScale, 2.25 * vScale);
+    };
```

Comment:
I'd do more simple:
```js
    PDFSlidesViewer.prototype.getDefaultScale = function(page) {
        var maxWidth = this.canvas.parentNode.clientWidth;
        var hScale = maxWidth / page.view[2];
        return hScale
    };
```

Reply:
I kept the vscale calculation in, because oftentimes it's smaller than the hscale (since the document is likely to be taller than it is wide, and on the other hand the screen is likely to be wider than it is tall)